rootfs/scheduler/resources/ingress.py

```diff
@@ -55,7 +55,7 @@ def create(self, ingress, namespace, hostname):
         return response
 
     def delete(self, namespace, ingress):
-        url = self.api("/namespaces/{}/ingresses/{}", namespace, ingress)
+        url = "/apis/extensions/v1beta1/namespaces/%s/ingresses/%s" % (namespace, ingress)
```

Comment:
good catch. I also just checked to confirm and this API endpoint is also the same for v1.5. We'll have to refactor the `self.api` function now that kubernetes is going with the aggregated API route, but this works for now. 👍 

I'll file an issue for this.